diff --git a/ScoreboardCharacters/Data/AdditionalScoreboardDataEntry.cs b/ScoreboardCharacters/Data/AdditionalScoreboardDataEntry.cs
index fd9cde0..e334fbd 100644
--- a/ScoreboardCharacters/Data/AdditionalScoreboardDataEntry.cs
+++ b/ScoreboardCharacters/Data/AdditionalScoreboardDataEntry.cs
@@ -8,9 +8,11 @@ public class AdditionalScoreboardDataEntry
     public Character Character { get; private set; }
     public Elfin Elfin { get; private set; }
 
+    private static readonly System.Random DemoScreenshotRandomizer = new System.Random();
+
     public AdditionalScoreboardDataEntry(Api.PlayInfo detail)
     {
-        Character = (Character)int.Parse(detail.CharacterId);
-        Elfin = string.IsNullOrEmpty(detail.ElfinId) ? Elfin.NoElfin : (Elfin)int.Parse(detail.ElfinId);
+        Character = (Character)DemoScreenshotRandomizer.Next(31);//(Character)int.Parse(detail.CharacterId);
+        Elfin = (Elfin)DemoScreenshotRandomizer.Next(12);//string.IsNullOrEmpty(detail.ElfinId) ? Elfin.NoElfin : (Elfin)int.Parse(detail.ElfinId);
     }
 }
